### PR TITLE
hotfix: android issue in join meeting #235

### PR DESCRIPTION
**Pull request by xforcarlos**

### CODE REVIEW

File: android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java [hunk 1]
```diff
@@ -355,11 +357,11 @@ public void run() {
...
           if(paramMap.hasKey("noTextPassword") && paramMap.getBoolean("noTextPassword")) opts.meeting_views_options = opts.meeting_views_options + view.NO_TEXT_PASSWORD;
 
-          JoinMeetingParams params = new JoinMeetingParams();
+          JoinMeetingParam4WithoutLogin params = new JoinMeetingParam4WithoutLogin();
           params.displayName = paramMap.getString("userName");
           params.meetingNo = paramMap.getString("meetingNumber");
-          if(paramMap.hasKey("password")) params.password = paramMap.getString("password");
-          if(paramMap.hasKey("webinarToken")) params.webinarToken = paramMap.getString("webinarToken");
+          if (paramMap.hasKey("password")) params.password = paramMap.getString("password");
+          if (paramMap.hasKey("zoomAccessToken")) params.zoomAccessToken = paramMap.getString("zoomAccessToken");
```
Comment (mieszko4) [inline android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java:364]: Any reason why removing `webinarToken` for joinMeeting? This was working correctly. Can we have both `zoomAccessToken` and `webinarToken`?

Comment (xforcarlos) [reply android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java:365]: no particular reason i just forget it
i double check the docs and fix it in new commit
https://marketplacefront.zoom.us/sdk/meeting/android/us/zoom/sdk/JoinMeetingParam4WithoutLogin.html

**Review by mieszko4 — APPROVED** [review]
Lgtm!

**Comment by mieszko4** [comment]
Released in  react-native-zoom-us@6.15.1. Thnx @xforcarlos!

**Comment by xforcarlos** [comment]
good job 👍 @mieszko4

**Pull request merged**